pandas/tests/groupby/test_grouping.py

```diff
@@ -203,6 +203,18 @@ def test_grouper_creation_bug(self):
         expected = s.groupby(level='one').sum()
         assert_series_equal(result, expected)
 
+        # GH25161
+        s = Series(np.arange(10),
+                   index=pd.date_range(
+                       "2018-11-26 16:17:43.51",
+                       periods=10, freq="1S"))
+        t0 = s.index[0]
+        base = t0.minute + (t0.second + t0.microsecond / 1e6) / 60
```

Comment:
Can you replace expressions like these with literal values? I realize this is copied directly from referenced issue but writing it this way makes the test more difficult to reason about than it needs to be

Reply:
Done! Thanks